benchmarks/lasso/solvers/cd.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+from scipy import sparse
+from numba import njit
+
+from numpy.linalg import norm
+
+from benchopt.base import BaseSolver
+
+
+@njit
+def st(x, mu):
+    if x > mu:
+        return x - mu
+    if x < - mu:
+        return x + mu
+    return 0
+
+
+class Solver(BaseSolver):
+    name = "cd"
+
+    install_cmd = None
+
+    def set_objective(self, X, y, lmbd):
+        self.X, self.y, self.lmbd = X, y, lmbd
+        # self.run(1)
```

Comment:
@tomMoral should we run 1 iteration to compile numba? Or is it neglectable ?

Reply:
yes do this to cache the compiled code